.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+install:
+  - pip install flake8
```

Comment:
Maybe change this to `-r test-requirements.txt` so we run the version of flake8 specified in the project?